pypdf/_page.py

```diff
@@ -27,9 +27,12 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+import json
```

Comment:
While this is only used for debugging purposes, do we really need this as a top-level import?